docs/crd.md

```diff
@@ -0,0 +1,50 @@
+# Restoring Deleted CRs
+
+When a Rook CR is deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources. If any data is still present in the cluster, Rook will refuse to delete the CR to ensure data is not lost. The operator will refuse to remove the finalizer on the CR until the underlying data is deleted.
+
+While it is good that the CRs will not be deleted and the underlying Ceph data and daemons continue to be available, the CRs will be stuck indefinitely in a Deleting state in which the operator will not continue to ensure cluster health. Upgrades will be blocked, further updates to the CRs are prevented, and so on. Since Kubernetes does not allow undeleting resources, the command below will allow repairing the CRs without even necessarily suffering cluster downtime.
+
+
+## Restore Command
+
+```bash
+kubectl rook-ceph restore-deleted <CRDName> [CRName]
+
+# `restore-deleted`: is the command to restore the deleted CR
+# `<CRDName>`: the CRD deleting you want to restore. Example: cephcluster, cephfilesystem, cephblockpool and so on.
+# `[CRName]`: is the name of the specific CR which you want to restore since there can be multiple CRs under the same CRD. Example let's say you multiple cephfilesystem stuck in deleting state in your cluster named `filesystem-1`, `filesystem-2` and you want to restore to store only `filesystem-2`. In this case you can pass the name of the specifc CR, like this: `restore-deleted cephfilesystem filesystem-2`.
```

Comment:
Shorter can be more concise to understand.
```suggestion
# `[CRName]`: is the name of the specific CR which you want to restore since there can be multiple instances under the same CRD. For example, if there are multiple CephFilesystems stuck in deleting state, a specific filesystem can be restored: `restore-deleted cephfilesystem filesystem-2`.
```